Applications/Analyze/test/subject01_adjusted_spring.osim

```diff
@@ -1,1611 +1,1444 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<OpenSimDocument Version="30503">
+<OpenSimDocument Version="30000">
```

Comment:
I can alternatively rename `reference_frame` to `parent_frame` everywhere within the model file.